core/src/main/java/org/apache/calcite/adapter/enumerable/RexImpTable.java

```diff
@@ -928,6 +933,10 @@ public MatchImplementor get(final SqlMatchFunction function) {
     }
   }
 
+  public TableValuedFunctionCallImplementor getTvfImplementor(final SqlOperator operator) {
+    return tvfImplementorMap.get(operator);
```

Comment:
Rename this method to `get` and pass in argument as type `SqlWindowTableFunction`.